packages/commerce-sdk-react/src/hooks/ShopperProducts/query.test.ts

```diff
@@ -12,13 +12,19 @@ import {
     waitAndExpectSuccess
 } from '../../test-utils'
 import * as queries from './query'
+import * as utilModuleApi from '../../utils'
 
 jest.mock('../../auth/index.ts', () => {
     const {default: mockAuth} = jest.requireActual('../../auth/index.ts')
     mockAuth.prototype.ready = jest.fn().mockResolvedValue({access_token: 'access_token'})
     return mockAuth
 })
 
+jest.mock('../../utils', () => {
+    const onClient = jest.requireActual('../../utils.ts')
+    return onClient
+})
```

Comment:
What's the point of a mock that just returns the actual module?

Reply:
Can we do this instead? Most the test mocks are using this pattern, so I'd suggest we keep it consistent
```
import {onClient} from '../../utils'

jest.mock('../../utiles', () => {
   const original = jest.requireActual('../../utils.ts')
  return {
      ...original,
     onClient: jest.mock()
  
  }
}

test('some-test' , () => {
    onClient.returnMockValue(false)
   ///rest of the test

})
```

Reply:
Or alternatively, I think this also works cleaner if you only have 1 test need to mock it.

```js
import utils from '../../utils'

// in your unit test
jest.spyOn(utils, onClient).mockReturnValue(false)
```